span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -229,15 +229,17 @@ private Event buildEvent(
     }
 
     // Jaeger service name
-    if (jaegerSpan.getProcess() != null && jaegerSpan.getProcess().getServiceName() != null) {
+    if (jaegerSpan.getProcess() != null && jaegerSpan.getProcess().getServiceName() != null
+        && !jaegerSpan.getProcess().getServiceName().isEmpty()) {
```

Comment:
Default value of ```service_name``` in ```Process``` is ```""``` not null. Let me know if we want this or not

Reply:
Yah its redundant, lets replace it with ` !StringUtils.isEmpty(aegerSpan.getProcess().getServiceName())`

Reply:
we can do - `!StringUtils.isEmpty(aegerSpan.getProcess().getServiceName())`